File: ambeth/Ambeth.Privilege/ambeth/model/IPrivilegeResult.cs
using System;
using System.Collections.Generic;
using De.Osthus.Ambeth.Annotation;
using De.Osthus.Ambeth.Merge.Model;
using De.Osthus.Ambeth.Model;
using System.Runtime.Serialization;

namespace De.Osthus.Ambeth.Privilege.Model
{
    public interface IPrivilegeResult
    {
        IObjRef Reference { get; }

        ISecurityScope SecurityScope { get; }

        PrivilegeEnum[] Privileges { get; }
    }
}

File: jambeth/jambeth-security/src/main/java/de/osthus/ambeth/privilege/model/IPrivilegeResult.java
package de.osthus.ambeth.privilege.model;

import de.osthus.ambeth.annotation.XmlType;
import de.osthus.ambeth.merge.model.IObjRef;
import de.osthus.ambeth.model.ISecurityScope;

@XmlType
public interface IPrivilegeResult
{
	IObjRef getReference();

	ISecurityScope getSecurityScope();

	PrivilegeEnum[] getPrivileges();
}
